examples/todo-mvc-react/src/App.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Todos } from './Todos';
+
+function App() {
+  return (
+    <>
+      <Todos />
+
+      <footer class="info">
+        <p>Double-click to edit a todo</p>
+        <p>
+          Template by <a href="http://sindresorhus.com">Sindre Sorhus</a>
+        </p>
+        <p>
+          Created by <a href="http://todomvc.com">you</a>
+        </p>
+        <p>
+          Part of <a href="http://todomvc.com">TodoMVC</a>
+        </p>
+      </footer>
```

Comment:
Moved `footer` from `index.html` to `App.jsx` in order to avoid showing it up for a moment before the App renders.